plugins/flytekit-polars/flytekitplugins/polars/sd_transformers.py

```diff
@@ -40,23 +40,32 @@ def encode(
         structured_dataset_type: StructuredDatasetType,
     ) -> literals.StructuredDataset:
         df = typing.cast(pl.DataFrame, structured_dataset.dataframe)
-
+        random_string = ctx.file_access.get_random_string()
```

Comment:
why do we need the random string here?

Reply:
I changed this back to the default `00000` parquet filename